src/Components/MinimalComponentModel.cs

```diff
@@ -124,6 +124,26 @@ public MinimalComponentModel(PermissionFlags permissions)
         /// </summary>
         public ComponentStateType State { get; set; }
 
+        /// <summary>
```

Comment:
Let's put these properties into a sub-class, named ComponentMetricsModel, and make a property of that type in this class called Metrics... I feel like this info can expand and get unwieldly